dbms/src/Storages/Page/V3/spacemap/SpaceMap.cpp

```diff
@@ -43,9 +42,6 @@ SpaceMapPtr SpaceMap::createSpaceMap(SpaceMapType type, UInt64 start, UInt64 end
     case SMAP64_STD_MAP:
```

Comment:
BTW, can we remove `RBTreeSpaceMap`? I don't think it is ever useful.

Reply:
Yes, we can remove it in another pr.